.github/workflows/development-environment.yml

```diff
@@ -16,11 +16,28 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        os: [ macos-11.0 ]
+        os: [ macos-latest ]
```

Comment:
This makes the jobs be pending less. I assume the Catalina pool is much larger.

Do you want me to test both OSes?

Reply:
Nah, we already know Catalina works and I'd say I don't care about regressions. We want to encourage people to be on latest.